Android/Google.Android.DataTransport/source/Xamarin.Google.Android.DataTransport.TransportApi/Xamarin.Google.Android.DataTransport.TransportApi.csproj

```diff
@@ -0,0 +1,51 @@
+<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
+  <PropertyGroup>
+    <AssemblyName>Xamarin.Google.Android.DataTransport.TransportApi</AssemblyName>
+
+    <MonoAndroidResourcePrefix>Resources</MonoAndroidResourcePrefix>
+    <MonoAndroidAssetsPrefix>Assets</MonoAndroidAssetsPrefix>
+    <AndroidUseLatestPlatformSdk>False</AndroidUseLatestPlatformSdk>
+    <AndroidUseIntermediateDesignerFile>True</AndroidUseIntermediateDesignerFile>
+    <AndroidResgenFile>Resources\Resource.designer.cs</AndroidResgenFile>
+
+    <TargetFrameworks>MonoAndroid90</TargetFrameworks>
+    <AndroidCodegenTarget>XAJavaInterop1</AndroidCodegenTarget>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <IsBindingProject>true</IsBindingProject>
+    <AndroidClassParser>class-parse</AndroidClassParser>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <PackageId>Xamarin.Google.Android.DataTransport.TransportApi</PackageId>
+    <PackageVersion>2.0.0</PackageVersion>
+    <Title>Xamarin.Google.Android.DataTransport.TransportApi</Title>
+    <PackageDescription>Bindings for Google.Android.DataTransport.TransportApi package</PackageDescription>
+    <Owners>Microsoft</Owners>
+    <Authors>Microsoft</Authors>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <PackageLicenseUrl>https://go.microsoft.com/fwlink/?linkid=2109280</PackageLicenseUrl>
+    <PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=2109279</PackageProjectUrl>
+    <RequireLicenceAcceptance>true</RequireLicenceAcceptance>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <PackageTags>xamarin, android, bindings, google, datatransport, transport, api </PackageTags>
+    <PackOnBuild>true</PackOnBuild>
```

Comment:
I would leave off the `<PackOnBuild>` in the .csproj as this just makes the builds in the IDE slower. We get NuGets from the `/t:pack` target.

Reply:
OK. Didn't know that.
Removed.

Reply:
I had that to ensure that nuget is built from IDE too. 
Few ms in IDE for 1 or 2 maybe 3 projects - I can bear with that.
I know we build and package nugets on CI, but this was simply to be sure it is packed.